docs/reference/modules/asset.md

```diff
@@ -0,0 +1,205 @@
+---
+extends: '@apostrophecms/module'
+---
+
+# `@apostrophecms/asset`
+
+**Alias:** `apos.asset`
+
+<AposRefExtends :module="$frontmatter.extends" />
+
+The `asset` module serves to organize, process, and output all project script and styling assets during the build process. In addition, it provides access to modify the project webpack configuration and exposes two CLI tasks for project building and webpack cache clearing. Options are passed through the creation of a `modules/@apostrophecms/asset/index.js` file.
```

Comment:
I would explicitly say "JavaScript and CSS assets." Helps people find this, search wise.

Reply:
Changed.